modules/sops/templates/default.nix

```diff
@@ -0,0 +1,102 @@
+{ config, pkgs, lib, options, ... }:
+with lib;
+with lib.types;
+with builtins;
+let
+  cfg = config.sops;
+  secretsForUsers = lib.filterAttrs (_: v: v.neededForUsers) cfg.secrets;
+in {
+  options.sops = {
+    templates = mkOption {
+      type = attrsOf (submodule ({ config, ... }: {
+        options = {
+          name = mkOption {
+            type = str;
+            default = config._module.args.name;
+            description = ''
+              Name of the file used in /run/secrets-rendered
+            '';
+          };
+          path = mkOption {
+            type = str;
+            default = "/run/secrets-rendered/${config.name}";
+          };
+          content = mkOption {
+            type = lines;
+            default = "";
+            description = ''
+              Content of the file
+            '';
+          };
+          mode = mkOption {
+            type = str;
+            default = "0400";
+            description = ''
+              Permissions mode of the rendered secret file in octal.
+            '';
+          };
+          owner = mkOption {
+            type = str;
+            default = "root";
+            description = ''
+              User of the file.
+            '';
+          };
+          group = mkOption {
```

Comment:
Can we set default group to `root`?
```
   error: attribute 'users' missing

       at /nix/store/pdi20jccn9ah62y3ny1dqwaqk869x995-source/modules/sops/templates/default.nix:47:23:

           46|             type = str;
           47|             default = config.users.users.${config.owner}.group;
             |                       ^
           48|             description = ''
Build Failed
```

Reply:
This is in line with the `sops.secrets` settings. Why is `config.users` missing? How do you use `sops-nix`?

Reply:
Oh, I see the error here, I'll submit a fix soon.